src/Events.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import Navbar from './components/Navbar';
+import Footer from './components/Footer';
+import EventsStyled from './EventsStyled';
+import code from './components/assets/image/homepage/code.jpg'
+
+function Events() {
+    return (
+        <React.Fragment>
+            <EventsStyled>
+                <Navbar />
+                <div className="norm-events">
+                    <div style={{ paddingBottom: "1em", height: "30em", overflow: "hidden", position: "relative" }}>
+                        <div className="">
+                            <img src={code} style={{ objectFit: "contain", width: "100%" }} />
```

Comment:
Codacy found an issue: [img elements must have an alt prop, either with meaningful text, or an empty string for decorative images.](https://app.codacy.com/gh/codeclannigeria/codeclannigeria-frontend/pullRequest?prid=6078282)